client/public/clientJS/adminWishCards.js

```diff
@@ -7,13 +7,19 @@ window.onload = function () {
 
 function unlockCardButtonClick(e) {
     const cardId = e.target.dataset.valueId;
+    const oldItemUrl = document.getElementById("oldWishItemUrl"+cardId);
+    const newItemUrl = document.getElementById("newWishItemUrl"+cardId);
+    // take new url if anything is isnide input box
+    let wishItemUrl = newItemUrl.value ? newItemUrl.value : oldItemUrl.href;
+    console.log(wishItemUrl);
     // get reference to element so we can call it inside callback
     const elementRef = e.target;
     $.ajax({
         type: 'PUT',
         url: '/wishcards/admin',
         data: {
           wishCardId: cardId,
+          wishItemURL: wishItemUrl,
         },
         success: function (response, textStatus, xhr) {
           showToast("Card status updated");
```

Comment:
errorhandling has been changed, so error should now be in response.responseJSON.error.msg